deepmd/pt_expt/infer/deep_eval.py

```
@@ -955,3 +993,177 @@ def get_model(self) -> torch.nn.Module:
+        if dp_am is None:
+            raise NotImplementedError(
+                "eval_descriptor is not supported for this model type "
+                f"({type(self._dpmodel).__name__})."
+            )
+        (
+            ext_coord_t,
+            ext_atype_t,
+            nlist_t,
+            mapping_t,
+            fparam_t,
+            aparam_t,
```

Ruff (0.15.9)

[warning] Unpacked variable `aparam_t` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)